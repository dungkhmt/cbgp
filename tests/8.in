50 50
50 80
14 3
49 42
36 15
19 11
27 6
47 17
47 2
21 5
28 7
34 29
32 26
37 4
20 5
29 14
4 5
32 13
43 0
6 22
49 36
18 20
43 10
32 10
6 2
24 0
41 6
28 1
36 41
48 24
32 25
45 4
30 9
20 23
38 6
24 7
35 45
49 39
9 4
9 12
15 0
25 9
5 8
26 11
3 0
40 15
34 16
46 39
44 26
32 33
31 9
47 7
37 45
27 28
32 11
29 3
24 2
2 3
35 37
34 14
32 9
27 1
22 14
3 16
13 10
42 18
18 5
17 7
42 5
11 16
42 15
12 0
43 19
30 23
27 36
36 1
26 4
32 31
11 3
43 33
18 23
6 3
1
30 5 71 2 60 3 57 6 29 21 4 4 63 11 66 2 22 22 38 13 36 8 48 7 36 11 37 9 52 11 26 1 49 8 64 3 18 5 37 7 15 6 0 0 57 16 16 7 33 2 39 11 31 23 70 4 68 2 53 9 29 20 39 14 45 8 37 10 50 9 25 22 76 0 27 22 42 32 4 0 28 2 64 12 22 1 34 7 40 32 25 25 2 0 63 5 30 2 21 0