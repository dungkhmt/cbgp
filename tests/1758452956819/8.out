(14, 3), (49, 42), (36, 15), (19, 11), (27, 6), (47, 17), (47, 2), (21, 5), (28, 7), (34, 29), (32, 26), (37, 4), (20, 5), (29, 14), (4, 5), (32, 13), (43, 0), (6, 22), (49, 36), (18, 20), (43, 10), (32, 10), (6, 2), (24, 0), (41, 6), (28, 1), (36, 41), (48, 24), (32, 25), (45, 4), (30, 9), (20, 23), (38, 6), (24, 7), (35, 45), (49, 39), (9, 4), (9, 12), (15, 0), (25, 9), (5, 8), (26, 11), (3, 0), (40, 15), (34, 16), (46, 39), (44, 26), (32, 33), (31, 9), (47, 7), (37, 45), (27, 28), (32, 11), (29, 3), (24, 2), (2, 3), (35, 37), (34, 14), (32, 9), (27, 1), (22, 14), (3, 16), (13, 10), (42, 18), (18, 5), (17, 7), (42, 5), (11, 16), (42, 15), (12, 0), (43, 19), (30, 23), (27, 36), (36, 1), (26, 4), (32, 31), (11, 3), (43, 33), (18, 23), (6, 3), 
numberIntersection=0 
minDistance=2.0 
sumDistance=585744.5504497506 
minAngle=0.010525927031725502 
sumAngle=43.41286264571561 
minEdgeLength=2.0 
sumEdgeLength=3253.1862994343824 

0.0000000000 2.0000000000 585744.5504497506 0.0105259270 43.4128626457 2.0000000000 3253.1862994344
0: (10, 7),
1: (196, 3),
2: (195, 12),
3: (184, 185),
4: (5, 194),
5: (0, 2),
6: (195, 200),
7: (193, 3),
8: (0, 79),
9: (7, 192),
10: (15, 191),
11: (182, 198),
12: (5, 9),
13: (17, 194),
14: (190, 200),
15: (10, 2),
16: (184, 200),
17: (191, 5),
18: (7, 4),
19: (181, 189),
20: (3, 5),
21: (0, 0),
22: (192, 200),
23: (3, 8),
24: (186, 3),
25: (10, 191),
26: (0, 198),
27: (198, 5),
28: (195, 5),
29: (187, 198),
30: (3, 10),
31: (7, 195),
32: (15, 196),
33: (11, 188),
34: (186, 200),
35: (0, 191),
36: (200, 0),
37: (2, 192),
38: (198, 198),
39: (4, 0),
40: (12, 4),
41: (200, 200),
42: (8, 2),
43: (14, 189),
44: (0, 200),
45: (0, 195),
46: (2, 0),
47: (193, 6),
48: (161, 6),
49: (6, 0),
10 7 196 3 195 12 184 185 5 194 0 2 195 200 193 3 0 79 7 192 15 191 182 198 5 9 17 194 190 200 10 2 184 200 191 5 7 4 181 189 3 5 0 0 192 200 3 8 186 3 10 191 0 198 198 5 195 5 187 198 3 10 7 195 15 196 11 188 186 200 0 191 200 0 2 192 198 198 4 0 12 4 200 200 8 2 14 189 0 200 0 195 2 0 193 6 161 6 6 0 
0: (10, 7), 1: (196, 3), 2: (195, 12), 3: (184, 185), 4: (5, 194), 5: (0, 2), 6: (195, 200), 7: (193, 3), 8: (0, 79), 9: (7, 192), 10: (15, 191), 11: (182, 198), 12: (5, 9), 13: (17, 194), 14: (190, 200), 15: (10, 2), 16: (184, 200), 17: (191, 5), 18: (7, 4), 19: (181, 189), 20: (3, 5), 21: (0, 0), 22: (192, 200), 23: (3, 8), 24: (186, 3), 25: (10, 191), 26: (0, 198), 27: (198, 5), 28: (195, 5), 29: (187, 198), 30: (3, 10), 31: (7, 195), 32: (15, 196), 33: (11, 188), 34: (186, 200), 35: (0, 191), 36: (200, 0), 37: (2, 192), 38: (198, 198), 39: (4, 0), 40: (12, 4), 41: (200, 200), 42: (8, 2), 43: (14, 189), 44: (0, 200), 45: (0, 195), 46: (2, 0), 47: (193, 6), 48: (161, 6), 49: (6, 0), 
numberIntersection=0 minAngle=0.010525927031725502 sumAngle=43.412862645953176 minEdgeLength=2.0 sumEdgeLength=3253.1862994343455 minDistance=2.0 sumDistance=585744.5504497477 
Time taken: 26.191 s
